Sources/OktaOidc/macOS/Internal/Tasks/OktaOidcBrowserTaskMAC.swift

```diff
@@ -29,8 +29,9 @@ class OktaOidcBrowserTaskMAC: OktaOidcBrowserTask {
          oktaAPI: OktaOidcHttpApiProtocol,
          redirectServerConfiguration: OktaRedirectServerConfiguration? = nil) {
         if let redirectServerConfiguration = redirectServerConfiguration {
-            redirectServer = OktaRedirectServer(successURL: redirectServerConfiguration.successRedirectURL,
-                                                port: redirectServerConfiguration.port ?? 0)
+            redirectServer = OktaRedirectServer.shared
```

Comment:
It's generally preferred to avoid using singletons wherever possible, particularly since it could result in the server remaining open / outliving the authentication process.

Perhaps instead the redirect server could be stored as a property on the `OktaOidcRestApi` object. That way the redirect server could be shared across multiple tasks, but will be deallocated when the RestApi object goes away.